portfolio.css

```diff
@@ -0,0 +1,308 @@
+html {
+	font-size: 10px;
+	font-family: 'Trebuchet MS';
+}
+
+* {
+	box-sizing: border-box;
+}
+
+h1, h2, h3, h4 {
+	margin: 0;
+}
+
+a {
+	text-decoration: none;
+	font-weight: bold;
+	text-shadow: 0 0 2px #26315f;
+	
+}
+
+a, header, .about, .portfolio, footer {
+	color: #9a4a32;
```

Comment:
this colour does not have enough contrast with the blue background for accessibility, you should change it to a lighter colour that stands out more